book/src/algorithm.md

```diff
@@ -379,7 +379,7 @@ of the criteria required for the audit.
 
 The AuditGraph is the graph of all audits for a particular package *name*.
 The nodes of the graph are [Version][]s and the edges are delta audits (e.g. `0.1.0 -> 0.2.0`).
-Each edge has a list of criteria is claims to certify, and dependency_criteria that the
+Each edge has a list of criteria it claims to certify, and dependency criteria that the

```

Comment:
Oh, I just noticed that I missed this section when doing my update, as the AuditGraph no longer has dependency criteria after my changes.